### PR TITLE
Plugin zookeeper testcase

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Please fill up those following information before send Pull request

Pull request NO: 2211 (https://github.com/apache/incubator-skywalking/pull/2211)
Agent repository URL: https://github.com/zhaoyuguang/incubator-skywalking
Testcases repository URL: https://github.com/zhaoyuguang/skywalking-agent-testcases

### CODE REVIEW

**Comment by ascrutae** [comment]
Could you add the `watcher` case in this scenario?

**Comment by zhaoyuguang** [comment]
Zookeeper Watcher often continues to listen after processing events, it seems to want to loop infinitely, so here is a way to improve the return method, your suggestion? @ascrutae

**Comment by wu-sheng** [comment]
I think @ascrutae is asking you to add a case to improve the watcher works. And showus how it works.

**Comment by ascrutae** [comment]
The target that I want to add `water` case is to verify if the trace that the user has process business logic in `watcher` is  broken, and I found that the expected data is miss that data.

**Comment by zhaoyuguang** [comment]
I modified testcase, please see if it is feasible. @ascrutae

File: zookeeper-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      componentId: 58
+      componentName: ''
+      isError: false
+      spanType: 'not null'
+      peer: 'not null'
+      peerId: 0
+      tags:
+      - {key: path, value: /path}
+      - {key: state, value: SyncConnected}
+      - {key: db.type, value: Zookeeper}
```
Comment (ascrutae) [inline zookeeper-scenario/config/expectedData.yaml]: The expected data miss the `/zookeeper/zookeeper-case` span

Comment (zhaoyuguang) [reply zookeeper-scenario/config/expectedData.yaml]: The '/zookeeper/zookeeper-case' has added.

**Review by ascrutae — DISMISSED** [review]

**Review by ascrutae — APPROVED** [review]

**Pull request merged**